ui/components/multichain/asset-picker-amount/asset-picker/index.scss

```diff
@@ -0,0 +1,19 @@
+.asset-picker {
+  display: flex;
+  align-items: center;
+  padding: 7px;
+  min-width: 80px;
+  height: 30px;
+  border-radius: 15px;
+  background-color: var(--color-background-alternative);
```

Comment:
Should be able to use style utility props here too dosc https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-box--docs